client/packages/programs/src/JsonForms/components/IdGenerator.tsx

```diff
@@ -19,14 +19,22 @@ import {
   FORM_GAP,
   FORM_LABEL_WIDTH,
 } from '@openmsupply-client/programs';
+import { usePatient } from '../../../../system/src/Patient/api';
 import { get as extractProperty } from 'lodash';
```

Comment:
```suggestion
import { usePatient } from '@openmsupply-client/system';
```

We split the project into multiple npm packages and the previous import only works because of the current setup. In general we want to import from the correct package... Unfortunately the TS languagserver is not smart enough to do this automatically...